### DISCUSSION TITLE
footer contactus

### DISCUSSION THREAD

**Discussion by SrushtiPatiL25** [smaranjitghose/awesome-portfolio-websites#1139] [Q&A]
In footer part, there is section for get in touch.
At present if I provide feedback it gets sent to someone what if want that feedback to be sent to me ?
which file and what should be modified for the same?
Kindly help.

**Comment by Soran-i** [comment]
I need some help setting this up too

**Reply by anushbhatia** [reply]
Let us write a documentation for that and put it up in the readme

**Reply by TheUnderLing** [reply]
> Let us write a documentation for that and put it up in the readme

Did you end up writing the documentation for this, im having trouble setting it up with netlify

**Reply by Soran-i** [reply]
So I got this to work. I can write documentation for this issue.

**Reply by TheUnderLing** [reply]
> So I got this to work. I can write documentation for this issue.

Please do, would love to have a look through it
